lib/app.js

```diff
@@ -49,6 +49,27 @@ app.get('/chords/:id', async (req, res)=>{
   }
 });
 
+app.post ('/chords', async(req, res)=>{
+  try{
+    const data = await client.query(`
+    INSERT INTO chords(
+      id,
```

Comment:
Remember with a POST you don't need to `INSERT` the id in your SQL query because SQL does this automatically for you.